### DISCUSSION TITLE
consult-ripgrep preview configuration

### DISCUSSION THREAD

**Discussion by hjudt** [minad/consult#881] [Q&A] [answered]
Hi,

I have deactivated previews because of performance issues with TRAMP locations but would like to use them on demand. Is there a way to configure activating previews on key press for consult-ripgrep results, similar to: `(consult-customize consult--source-bookmark :preview-key '("M-." :debounce 1.5 any))` ?

**Comment by geza-herman** [comment]
If you haven't already, check out the [wiki](https://github.com/minad/consult/wiki), there are code snippets related your question.

**Comment by minad** [comment, marked as answer]
See https://github.com/minad/consult#live-previews for the relevant documentation.
